lona/compat.py

```diff
@@ -14,6 +16,7 @@
 def get_client_version():
     # TODO: remove in 2.0
 
+    lona.warnings.remove_2_0()
```

Comment:
All warnings in this module are incorrect because they run no matter what the user does, and are meant to remain compatible with old API. The `TODO: remove in 2.0` is only there so we don't forget to remove the functions before releasing Lona 2